crates/client/rpc/src/versions/admin/v0_1_0/api.rs

```diff
@@ -0,0 +1,16 @@
+use m_proc_macros::versioned_rpc;
+use mp_transactions::BroadcastedDeclareTransactionV0;
+
+use jsonrpsee::core::RpcResult;
+use starknet_core::types::DeclareTransactionResult;
+
+/// This is an admin method, so semver is different!
```

Comment:
Why? These are not tied by a common spec and we might want to make changes to admin methods which constitute a version change: we cannot do this if we tie admin and other rpc methods under the same semver.

Reply:
No, that's not what I meant. The madara binary has a version, and since this is a madara-specific API, it would make sense to have them be the same. Major or breaking changes would reflect naturally with the binary.

Actually, it doesn't look like the binary is versioned. I don't see anything in the `--help` output and there's no `--version` flag. Anyway, _assuming_ the madara node is versioned, it's admin RPC version should be the same. Again, I'm not talking about other RPC methods.